go/systests/team_reset_test.go

```diff
@@ -791,7 +859,6 @@ func TestTeamListAfterReset(t *testing.T) {
 }
 
 func TestTeamAfterDeleteUser(t *testing.T) {
-	t.Skip()
```

Comment:
I'm still hesitant to re-enable it, it passes locally but apparently sometimes it will fail on CI. I'm looking into it.